src/richie/apps/core/templates/richie/three_columns_33.html

```diff
@@ -0,0 +1,52 @@
+{% extends "richie/base.html" %}
+{% load cms_tags %}
+
+{% block head_title %}
+    {% page_attribute "page_title" %}
+{% endblock head_title %}
+
+{% block meta_opengraph_contextuals %}
+    {% page_attribute "page_title" as page_title %}
+    <meta property="og:title" content="{{ page_title|truncatechars:65 }}">
+    <meta property="og:url" content="{{ SITE.web_url }}{{ current_page.get_absolute_url }}">
+{% endblock meta_opengraph_contextuals %}
+
+
+{% block subheader_content %}
+<div class="subheader__container">
+    <h1 class="subheader__title">
+        {% render_model request.current_page "title" %}
+    </h1>
+</div>
+{% endblock subheader_content %}
+
+
+{% block content %}
+<div class="multiple-columns">
+
+    <div class="multiple-columns__w33 multiple-columns__short">
+    {% block column_first %}
+        {% with header_level=2 %}
+            {% placeholder "maincontent" %}
+        {% endwith %}
+    {% endblock column_first %}
+    </div>
+
+    <div class="multiple-columns__w33 multiple-columns__short">
+    {% block column_second %}
+        {% with header_level=3 %}
+            {% placeholder "secondcontent" %}
+        {% endwith %}
+    {% endblock column_second %}
+    </div>
+
+    <div class="multiple-columns__w33 multiple-columns__short">
+    {% block column_third %}
+        {% with header_level=3 %}
+            {% placeholder "thirdcontent" %}
+        {% endwith %}
+    {% endblock column_third %}
+    </div>
+
+</div>
+{% endblock content %}
```

Comment:
In order to improve maintainability and ease the creation of new multi columns template, I wondered if we could factorise these templates. Unfortunately it seems too complicated to create columns through a iterator as we have to name uniquely `block` and `placeholder`.

But maybe we could create a `multiple_columns.html` template and extend all those new templates from it in order to override only block content in each templates? WDYT?

Reply:
Technically there is no problem to do this i think. I will try.